src/pages/Admin/Venue/VenueDetailsForm.tsx

```diff
@@ -0,0 +1,351 @@
+          ? history.push(`/admin/${createUrlSafeName(venueId ?? vals.name)}`)
+          : history.push(`/admin`);
+      } catch (e) {
+        setFormError(true);
+        Bugsnag.notify(e, (event) => {
+          event.addMetadata("Admin::Venue::DetailsForm::onSubmit", {
+            venueId,
+            vals,
+          });
+        });
+      }
+    },
+    [
+      user,
+      formError,
+      venueId,
+      history,
+      sovereignVenueId,
+      sovereignVenue,
+      dispatch,
+    ]
+  );
+
+  const iconsMap = useMemo(
+    () => ({
+      [iconPositionFieldName]: {
+        width: PLAYA_VENUE_SIZE,
+        height: PLAYA_VENUE_SIZE,
+        top: defaultValues?.placement?.y ?? 0,
+        left: defaultValues?.placement?.x ?? 0,
+      },
+    }),
+    [defaultValues]
+  );
+
+  const onBoxMove: ExtractProps<
+    typeof PlayaContainer
+  >["onChange"] = useCallback(
+    (val) => {
+      if (!(iconPositionFieldName in val)) return;
+      const iconPos = val[iconPositionFieldName];
+      setValue("placement", {
+        x: iconPos.left,
+        y: iconPos.top,
+      });
+    },
+    [setValue]
+  );
+
+  useEffect(() => {
+    if (!previous || isTruthy(state.templatePage)) return;
+
+    previous();
+  }, [previous, state.templatePage]);
+
+  if (!state.templatePage) {
+    // In reality users should never actually see this, since the useEffect above should navigate us back to ?page=1
+    return <>Error: state.templatePage not defined.</>;
+  }
+
+  const isAdminPlaced =
+    state.detailsPage?.venue?.placement?.state ===
+    VenuePlacementState.AdminPlaced;
+  const placementAddress = state.detailsPage?.venue?.placement?.addressText;
+
+  // @debt refactor any needed styles out of Admin.scss (eg. toggles, etc) and into DetailsForm.scss/similar, then remove the admin-dashboard class from this container
+  return (
+    <div className="page page--admin admin-dashboard">
+      <div className="page-side page-side--admin">
+        <div className="page-container-left page-container-left">
+          <div className="page-container-left-content">
+            <VenueDetailsSubForm
+              venueId={venueId}
+              setValue={setValue}
+              state={state}
+              previous={previous}
+              sovereignVenue={sovereignVenue}
+              isSubmitting={isSubmitting}
+              register={register}
+              watch={watch}
+              onSubmit={onSubmit}
+              editing={!!venueId}
+              formError={formError}
+              setFormError={setFormError}
+              control={control}
+              handleSubmit={handleSubmit}
+              errors={errors}
+              setError={setError}
+            />
+          </div>
+        </div>
+      </div>
+      {IS_BURN && (
+        <div className="page-side preview" style={{ paddingBottom: "20px" }}>
+          <h4
+            className="italic"
+            style={{ textAlign: "center", fontSize: "22px" }}
```

Comment:
please use `className` for styling the title

Reply:
[OOS] Stance is to not fix all issues in this form to just get replaced by Admin v3 ones. Besides, `IS_BURN` code blocks will be removed by merge from staging.

Reply:
[Note]
Lines in this file show up as added due to the rename of it.
Most of them have remained unchanged from their previous iteration.
Only minimal changes made in the actual code due to the display of errors.